logback-core/src/main/java/ch/qos/logback/core/net/SyslogAppenderBase.java

```diff
@@ -36,6 +38,7 @@
 
   Layout<E> layout;
   String facilityStr;
+  String hostname;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2371638626/issues/source?bid=3222618&fileBranchId=3222706#l41)